tracer/src/Datadog.Trace/DiagnosticListeners/AspNetCoreDiagnosticObserver.cs

```diff
@@ -735,6 +793,45 @@ internal struct BeforeActionStruct
             public RouteData RouteData;
         }
 
+        /// <summary>
+        /// https://github1s.com/dotnet/aspnetcore/blob/v3.0.3/src/Mvc/Mvc.Core/src/Controllers/ControllerActionDescriptor.cs
+        /// </summary>
+        [DuckCopy]
+        internal struct ControllerActionDescriptorStruct
+        {
+            [Duck]
+            public MethodInfo MethodInfo;
+
+            [Duck]
+            public TypeInfo ControllerTypeInfo;
+        }
+
+        /// <summary>
+        /// https://github1s.com/dotnet/aspnetcore/blob/v3.0.3/src/Mvc/Mvc.RazorPages/src/CompiledPageActionDescriptor.cs
+        /// </summary>
+        [DuckCopy]
+        internal struct CompiledPageActionDescriptorStruct
+        {
+            [Duck]
+            public IEnumerable HandlerMethods;
+
+            [Duck]
+            public TypeInfo HandlerTypeInfo;
+        }
+
+        /// <summary>
+        /// https://github1s.com/dotnet/aspnetcore/blob/v3.0.3/src/Mvc/Mvc.RazorPages/src/Infrastructure/HandlerMethodDescriptor.cs
```

Comment:
```suggestion
        /// https://github.com/dotnet/aspnetcore/blob/v3.0.3/src/Mvc/Mvc.RazorPages/src/Infrastructure/HandlerMethodDescriptor.cs
```

Reply:
Fixed in adcb1b8cd07a697feb8e891a1a4f74b39b6d5d5a